app/main.py

```diff
@@ -1,8 +1,41 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons_list = [Person(person["name"], person["age"])for person in people]
+    for person in people:
+        iterable_person = Person.people[person["name"]]
+        if "wife" in person and person["wife"] and person["wife"] in persons_list is not None:
+            iterable_person.wife = Person.people[person["wife"]]
+        elif "husband" in person and person["husband"] and person["husband"] in persons_list is not None:
```

Comment:
Similarly, the condition `person['husband'] in persons_list is not None` is incorrect. You should check if `person['husband']` is a key in `Person.people` to verify if the husband exists in the list of created `Person` objects.